Create process/workflow proposal doc (process council) (#5120)
newrelic/docs-website: src/content/docs/style-guide/writing-docs/writer-workflow/process-changes.mdx

---
title: Proposing process or workflow changes
topics:
  - Tech writer style guide
  - Processes and procedures
---

Use this document to learn how to propose changes to the process or workflow in a collaborative manner.

> 

Reviewer: Is this doc for any team's use or primarily for our team? Also, we used to have a fairly strong mandate to avoid "ing" verb form for doc titles, primarily in the hopes of better SEO. (For example, readers would be more likely to search for "delete account" rather than "deleting an account.") Not sure if that's still a requirement. Style guide has changed a lot, so you may want to check on that.

## Proposing a change

> Do you have an idea you want to try out? It can be something big or something little or even something you haven't figured out how to solve yet. It can be a small suggestion such as using conventional commits, or a large scale change such as automating PR reviews. If so, follow these steps so the team can collaborate on the issue or suggestion.

Reviewer 2: ```suggestion
Do you have an idea you want to try out? It can be something big or something little or even something you haven't figured out how to solve yet. It can be a small suggestion such as using conventional commits, or a large-scale change such as automating PR reviews. If so, follow these steps so the team can collaborate on the issue or suggestion.
```

Reviewer 2: Mostly I just wanted to try out the suggestion thing. Slick!

> 1. Write your idea as a GitHub issue. The issue should have a brief overview, the actual proposal, and any relevant action items. See [this issue](https://github.com/newrelic/docs-website/issues/4553) for an example. If all you have is an issue and no proposal, that is fine too. The team can collaborate to solve it in the next step.

Reviewer 2: ```suggestion
1. Write your idea as a GitHub issue. The issue should have a brief overview, the actual proposal, and any relevant action items. See [this issue](https://github.com/newrelic/docs-website/issues/4553) for an example. If all you have is an issue and no proposal, that's fine too. The team can collaborate to solve it in the next step.
```

Reviewer: Style guide says to avoid things like "this issue" as the link text to avoid any suspicion of click bait. https://docs.newrelic.com/docs/style-guide/formatting/hyperlinks/  Better practice is to tell the reader exactly where the link goes.

> 2. Once you've done that, let other people know about it in Slack. Try to invite both positive and negative feedback. Not everyone may love a suggestion in it's first iteration. Avoid team-wide mentions unless it is a change that will drastically affect the entire team.

Reviewer 2: ```suggestion
2. Once you've done that, let other people know about it in Slack. Try to invite both positive and negative feedback. Not everyone may love a suggestion in its first iteration. Avoid team-wide mentions unless it is a change that will drastically affect the entire team.
```

> 2. Once you've done that, let other people know about it in Slack. Try to invite both positive and negative feedback. Not everyone may love a suggestion in it's first iteration. Avoid team-wide mentions unless it is a change that will drastically affect the entire team.

Reviewer: Do you need "try to"?

> 3. Feedback and conversations over the suggestion should happen in the issue itself. This will reduce Slack fatigue and keep all the information organized in one place. 

Reviewer 2: I think you should lose the "should" and make this more directive. :)

4. Once you've got feedback, finalize the proposal in the form of a pull request against our style guide or project management handbook. 

> 5. Let the team know you've submitted the final proposal in either Slack or a team meeting. The team has 7 days to continue collaborating on it or dig their heels in. If there are still conversations ongoing after 7 days, try to resolve those and figure out why there is friction. 

Reviewer 2: I think it would be good to clarify "dig their heels in". :) Does this just mean "reject the proposal"?

Reviewer 2: why not "one week" instead of "7 days"? What if the week includes holidays? Should we just say "5 business days"?

Reviewer: Also follow style guidelines re: when to spell out numbers

Reviewer: For me, "dig their heels in" implies that you are not necessarily welcoming healthy debate. That seems opposite of the earlier step about inviting both positive and negative feedback or alternative ideas.

> 6. Once the 7 days are over or there is a team-wide concensus, merge the pull request in. It is now a formal team process!

Reviewer 2: ```suggestion
6. Once the 7 days are over or there is a team-wide consensus, merge the pull request in. It's now a formal team process!
```

Reviewer 2: Maybe instead of team-wide consensus, we should use a consent model. It flips things a little. Instead of getting everyone to explicitly buy in to a thing before moving forward, we move forward unless there are strong objections. This lets us move forward in the face of silence or low engagement. Makes it easier to try things out and change things later if it doesn't work.

Reviewer: same comment re: spelling out numbers
